samza-test/src/test/java/org/apache/samza/test/operator/SessionWindowApp.java

```diff
@@ -50,7 +50,7 @@ public void init(StreamGraph graph, Config config) {
 
     pageViews
         .filter(m -> !FILTER_KEY.equals(m.getUserId()))
-        .window(Windows.keyedSessionWindow(PageView::getUserId, Duration.ofSeconds(3)))
+        .window(Windows.keyedSessionWindow(PageView::getUserId, Duration.ofSeconds(3), null, null))
```

Comment:
Would be nice to provide actual serdes as params since these are meant as examples (however undiscoverable).